interface/modules/custom_modules/oe-module-faxsms/src/Controller/RCFaxClient.php

```
@@ -744,6 +782,31 @@ public function getUser(): string
         return json_encode([$u['fname'], $u['lname'], $u['fax']]);
     }
 
+    /**
+     * @return string
+     */
+    public function getNotificationLog(): string
+    {
+        $type = $this->getRequest('type');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$type'. (undefined)

(UnusedLocalVariable)